src/_includes/homepage/spotlight.njk

```diff
@@ -1,9 +1,13 @@
 <div class="c-homepage-card c-homepage-card--spotlight">
-	<img
-		class="c-homepage-card__spotlight-photo"
-		alt="Photo of {{ spotlight.name }}."
-		onclick="location.href='/spotlight/{{ spotlight.name | slugify | url }}'"
-		src="img/spotlight/{{ spotlight.image }}" />
+	{%
+		set avatar = {
+			cssClass: "c-homepage-card__spotlight__avatar",
```

Comment:
Can we reintroduce `c-homepage-card__spotlight-photo` as a class name? We try not to chain Elements within Elements with [BEM](http://getbem.com/introduction/).